src/Microsoft.ML.TimeSeries/PolynomialModel/PolynomialModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// a general polynomial model
+    /// </summary>
+    internal sealed class PolynomialModel : AbstractPolynomialModel
+    {
+        public PolynomialModel(IReadOnlyList<double> coeffs)
+            : base(coeffs)
+        {
+        }
```

Comment:
This constructor is never called, and so this class is actually never used (it seems the only `AbstractPolynomialModel` you use is actually the Linear Model).

Is this intended? Perhaps consider removing this? Or is there a reason to keep it?

Reply:
Yes, currently we are only using the LinearModel. But by design, the least square can also accept models more complex than the linear model, so I did the abstraction.